### ISSUE TITLE
Getting rid of the script directory

### ISSUE THREAD

**Issue by mozinator** [overtone/overtone#372]
Who uses the scripts from the script directory ? 

Without any instructions on how to use those scripts, maybe its better to move those scripts to overtone-contrib ?

**Comment by hlolli** [comment]
I agree, this seems to be part of @rosejn's own setup. I think this dir should be removed.
